Cargo.toml

```diff
@@ -7,15 +7,30 @@ authors = ["Gear Technologies"]
 
 [dependencies]
 gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
-codec = { package = "parity-scale-codec", version = "3.2.1", default-features = false }
-primitive-types = { version = "0.12.1", default-features = false, features = ["scale-info"] }
-scale-info = { version = "2.3.0", default-features = false }
 ft-main-io = { git = "https://github.com/gear-dapps/sharded-fungible-token.git", tag = "0.1.2" }
 ft-logic-io = { git = "https://github.com/gear-dapps/sharded-fungible-token.git", tag = "0.1.2" }
+async-trait = "0.1.60"
+escrow-io = { path = "io" }
 
 [build-dependencies]
-gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+escrow-io = { path = "io" }
```

Comment:
Do we need to double dependency here if is it in `[dependencies]`?

Reply:
> Do we need to double dependency here if is it in `[dependencies]`?

Yes